lib/modules/manager/tekton/readme.md

```diff
@@ -13,16 +13,17 @@ Tasks and Pipeline definitions can also live outside the Kubernetes cluster and
 The `tekton` manager focuses on providing updates to Tekton resource references.
 
 Right now, Renovate's Tekton manager only supports references that are [Bundles](https://tekton.dev/docs/pipelines/tekton-bundle-contracts/).
-See the [`tektoncd/resolution` project on GitHub](https://github.com/tektoncd/resolution) for the different kinds of Tekton references.
+See the [docs](https://tekton.dev/docs/pipelines/resolution/) for the different kinds of Tekton references and their corresponding resolvers.
 
 ### Using a Tekton Bundle reference
 
-There are two ways to use a Tekton Bundle reference:
+There are three ways to use a Tekton Bundle reference:
 
-1. Via the [`tektoncd/resolution` project](https://github.com/tektoncd/resolution)
-2. Via the `taskRun.spec.taskRef.bundle` and the `pipelineRun.spec.pipelineRef.bundle` attributes
+1. Via the [Bundles Resolver](https://tekton.dev/docs/pipelines/bundle-resolver/)
+2. Via the [`tektoncd/resolution` project](https://github.com/tektoncd/resolution)
+3. Via the `taskRun.spec.taskRef.bundle` and the `pipelineRun.spec.pipelineRef.bundle` attributes
```

Comment:
```suggestion
1. Via the [Tekton Bundles Resolver](https://tekton.dev/docs/pipelines/bundle-resolver/)
1. Via the [`tektoncd/resolution` project](https://github.com/tektoncd/resolution)
1. Via the `taskRun.spec.taskRef.bundle` and the `pipelineRun.spec.pipelineRef.bundle` attributes
```

We can just use `1.` and let the Markdown renderer order it for us. That way we can't get the numbering wrong. 😉

And we can use a better link name.